server.py

```diff
@@ -8,7 +9,35 @@ def desk():
     return render_template("index.html")
 
 
-# def ask(word):#
+@app.route("/tagged/word/%string word%")
+def req():
+    # return tagged word#
+    data = request.args.get('word', {}, type=str)
+    return jsonify(dict(results=str(data)))
+
+
+@app.route("/tagged/sentence/")

```

Comment:
Add some parameter here with `str` type and write it down into function arguments.
```suggestion
@app.route("/tagged/sentence/<string:sentence>")
```
Do the same with all functions and methods you see in this code.